testsuite/test-submit.sh

```diff
@@ -21,15 +21,18 @@ sparktests=y
 stormtests=y
 zookeepertests=y
 
+# Toggle y/n for different test types
+# standardtests: will run with local disk
+# nolocaldirtests: will run without local disk
+# dependencytests: check dependencies
```

Comment:
could we just say using or not using MAGPIE_NO_LOCAL_DIR.  "local disk" is not really correct.  Otherwise, basics of everything look fine.